command/agent_test.go

```diff
@@ -301,6 +305,7 @@ func TestAgent_HostBasedIDs(t *testing.T) {
 		cmd := &AgentCommand{
 			args: []string{
 				"-data-dir", tmpDir,
+				"-bind", "127.0.0.1",
```

Comment:
Got it - this makes sense then. The bind argument isn't required for all cases, but it will fail if it can't find a private IP to advertise, so it makes sense you had to set this now that the check moved.